static/app/views/eventsV2/breadcrumb.tsx

```diff
@@ -12,9 +12,16 @@ type Props = {
   location: Location;
   organization: Organization;
   event?: Event;
+  isHomepage?: boolean;
 };
 
-function DiscoverBreadcrumb({eventView, event, organization, location}: Props) {
+function DiscoverBreadcrumb({
+  eventView,
+  event,
+  organization,
+  location,
+  isHomepage,
+}: Props) {
```

Comment:
nit: github wont let me highlight the section, but just below this I think the `homepage=true` query param is leaking into `discoverTarget`. Not a big deal since I don't see this having any ui impact but I think we can probably clean it up somewhere between lines 31 - 35, or update the `eventView.generateBlankQueryStringObject()` function.

Reply:
Good call! I just wrapped the `...location.query` with an omit to drop the `homepage` param